package.json

```diff
@@ -87,5 +78,12 @@
       "soljson.js"
     ],
     "parser": "babel-eslint"
+  },
+  "dependencies": {
```

Comment:
since the build step produces a single `.js` file, there should be no runtime dependencies.